app/assets/javascripts/beak/widgets/accessibility/keybind.coffee

```diff
@@ -0,0 +1,64 @@
+import { isMac } from "./utils.js"
+import { KeyCombo } from "./key-combo.js"
+
+export class Keybind
+  # id: string
```

Comment:
`string` here, but `String` down below (and throughout the rest of this codebase)